app/javascript/searchable.vue

```diff
@@ -43,6 +44,19 @@ export default {
       return dayjs(this.searchable.updated_at).format(
         'YYYY年MM月DD日(dd) HH:mm'
       )
+    },
+    summary() {
+      const word = this.word
+      const wordsPattern = word.replaceAll(/[\s+]/g, '|')
```

Comment:
```suggestion
      const wordsPattern = word.replaceAll(/\s+/g, '|')
```

`[\s+]`は`\s+`の書き間違いかな?と思いました🤔